packages/telemetry/src/sendTelemetry.ts

```diff
@@ -136,7 +137,30 @@ const buildPayload = async () => {
   let payload: Record<string, unknown> = {}
   let project
 
-  const argv = require('yargs/yargs')(process.argv.slice(2)).argv
+  const processArgv = [...process.argv]
+
+  // On windows process.argv may not return an array of strings.
+  // It will look something like [a,b,c] rather than ["a","b","c"] so we must stringify them before parsing as JSON
+  // "os.type()" returns 'Windows_NT' on Windows. See https://nodejs.org/docs/latest-v12.x/api/os.html#os_os_type.
+  if (os.type() === 'Windows_NT') {
```

Comment:
I realize this package already used `os.type`, but the rest of the codebase seems to use `process.platform`. Any benefit of using one or the other?

Reply:
Not immediately aware of anything specific other than if we use `process.platform` it would be one less require?

Reply:
I probably didn't realize there even was a `process.platform`! Is there anything else telemetry uses from that `os` lib that isn't available through `process`? 

Also the strings that `process.platform` returns may be different than what's returned by `os`? We would want to either convert them to be the same, or update the existing entries in the telemetry database to match, so we don't have different strings for the same platform.

Reply:
I think that `os` is only used for the is windows conditionals. 

The actual payload that's sent as telemetry is constructed from data from the `envinfo` package. That would mean no changes to the actual payload and so no data discontinuities. Unless I'm mistaken?

Reply:
Sweet! Yeah let's axe it.

Reply:
I found this https://stackoverflow.com/a/61744810/88106
Seems to suggest `os.type()` might be more reliable in VMs

Reply:
The node docs https://nodejs.org/api/os.html#osplatform for `os.platform()` state that "The return value is equivalent to [process.platform](https://nodejs.org/api/process.html#processplatform)."

Although not sure if that SO comment was highlighting that that isn't reliable when operating in VMs.

Reply:
Turns out I was a genius all along! 😅